libraries/WiFi/src/WiFiClient.cpp

```diff
@@ -23,7 +23,7 @@
 #include <lwip/netdb.h>
 #include <errno.h>
 
-#define WIFI_CLIENT_DEF_CONN_TIMEOUT  (3)
+#define WIFI_CLIENT_DEF_CONN_TIMEOUT  (3000)
```

Comment:
The name should include the time scale to avoid such issues eg `WIFI_CLIENT_DEF_CONN_TIMEOUT_MS`

Reply:
Done